external/radiation/python/getdata.py

```diff
@@ -0,0 +1,240 @@
+import numpy as np
+from config import *
+import os 
+import xarray as xr 
+import warnings
+
+
+def random_numbers(LOOKUP_DIR, me):
+    data_dict = {}
+    # File names for serialized random numbers in mcica_subcol
+    if me == 0:
+        sw_rand_file = os.path.join(LOOKUP_DIR, "rand2d_sw.nc")
+    else:
+        sw_rand_file = os.path.join(LOOKUP_DIR, "rand2d_tile" + str(me) + "_sw.nc")
+    lw_rand_file = os.path.join(LOOKUP_DIR, "rand2d_tile" + str(me) + "_lw.nc")
+    data_dict['sw_rand'] = xr.open_dataset(sw_rand_file)['rand2d'].values
+    data_dict['lw_rand'] = xr.open_dataset(lw_rand_file)['rand2d'].values
+    del(sw_rand_file, lw_rand_file)
+
+    return data_dict 
+
+def lw(LOOKUP_DIR):
+        ## file names needed in lwrad()
+    
+    lw_dict = {}
+    dfile = os.path.join(LOOKUP_DIR, "totplnk.nc")
+    pfile = os.path.join(LOOKUP_DIR, "radlw_ref_data.nc")
+    lw_dict['totplnk'] = xr.open_dataset(dfile)["totplnk"].values
+    lw_dict['preflog'] = xr.open_dataset(pfile)["preflog"].values
+    lw_dict['tref'] = xr.open_dataset(pfile)["tref"].values
+    lw_dict['chi_mls'] = xr.open_dataset(pfile)["chi_mls"].values
+    del(dfile, pfile)
```

Comment:
I think you can probably delete all the manual `del` calls in this file. I'm not sure why they are needed.

Reply:
removed.